src/ploneintranet/activitystream/browser/activity_provider.py

```diff
@@ -144,10 +136,10 @@ def text(self):
         return text
 
     def is_attachment_supported(self):
-        return IAttachmentStoragable is not None
+        return True
 
     def is_preview_supported(self):
-        return IDocconv is not None
+        return True
 
```

Comment:
I think the point of these methods in plonesocial was that there were a variety of 'providers', some of which may or may not support previews and attachments. Having said that, I think that previews are pretty much a requirement for everything in ploneintranet, so this may indeed no longer be relevant. @gyst / @pysailor any thoughts there?